### PR TITLE
Documentation: Generate Dynamic Links

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr includes some python code to generate examples checkout commands and liqoctl download link automatically.

### CODE REVIEW

**Comment by frisso** [comment]
Cool!

File: docs/installation/liqoctl.md [hunk 1]
```diff
@@ -81,18 +69,30 @@ Make sure `/usr/local/bin` is in your `PATH` environment variable.
...
 
-Alternatively, you can download *liqoctl* from the [Liqo releases](https://github.com/liqotech/liqo/releases/) page on GitHub.
+Alternatively, you can manually download *liqoctl* from the [Liqo releases](https://github.com/liqotech/liqo/releases/) page on GitHub.
+
+(InstallationLiqoctlFromSource)=
+
+## Install liqoctl from source
+
+You can install *liqoctl* building it from source.
+To do it, clone the Liqo repository, build the *liqoctl* binary, and move it to a file location in your system `PATH`:
```
Comment (giorio94) [inline docs/installation/liqoctl.md:86]:
```suggestion
To do so, clone the Liqo repository, build the *liqoctl* binary, and move it to a file location in your system `PATH`:
```

File: docs/installation/liqoctl.md [hunk 2]
```diff
@@ -81,18 +69,30 @@ Make sure `/usr/local/bin` is in your `PATH` environment variable.
...
+
+```bash
+git clone https://github.com/liqotech/liqo.git
+cd liqo
+make ctl
+mv ./liqoctl somewhere/in/your/path
```
Comment (giorio94) [inline docs/installation/liqoctl.md:92]:
```suggestion
mv liqoctl /usr/local/bin/liqoctl
```

Would be consistent with the above instructions.

File: docs/conf.py [hunk 3]
```diff
@@ -95,3 +97,72 @@
...
+def __is_sem_version(version: str) -> bool:
+    if version == 'stable':
+        return True
+    return semver.VersionInfo.isvalid(version)
+
+# generate_liqoctl_version_warning generates the liqoctl version warning when the documentation is not for a released version.
+def generate_liqoctl_version_warning() -> str:
+    if not 'github_version' in html_context or not __is_sem_version(html_context['github_version']):
+        return "```{warning}\n\
+The liqoctl installation instruction will install the latest liqoctl released version, that may have different behavior than the one described. If you want to use the latest liqoctl version [build it from source](InstallationLiqoctlFromSource).\n\
```
Comment (giorio94) [inline docs/conf.py:156]:
```suggestion
The following instructions will guide you through the installation of the **latest stable version of *liqoctl***, which might exhibit a different behavior compared to the one shown in the rest of this documentation. If you want to use the latest *liqoctl* version, [build it from source](InstallationLiqoctlFromSource).\n\
```

Seems a bit clearer to me, but let me know your thoughts.

**Review by giorio94 — APPROVED** [review]

**Pull request merged**